config/config_test.go

```diff
@@ -257,6 +257,14 @@ func Test_Defaults(t *testing.T) {
 			path:          "Pool.MaxTxDataBytesSize",
 			expectedValue: 30000,
 		},
+		{
+			path:          "Pool.MinGasPrice",
+			expectedValue: uint64(1000000000),
```

Comment:
if I am not wrong the L2GasPriceSuggester has the param "DefaultGasPriceWei" that is the min GasPrice suggested. It is stored in the pool.gas_price table if the gas price calculated by the GasPricerSuggester is minor than this value. In this case it's not needed to have this MinGasPrice param as this min value is manager by the GasPricerSuggester. If we keep this MinGasPrice value and it is not equal to the GasPricerSuggester we can have issues or misunderstanding in the behaviour of the min gas price management

Reply:
you are right @agnusmor It is redundant